### ISSUE TITLE
Is there a way to use strings for the custom tooltip labels?

### ISSUE THREAD

**Issue by BlackArbsCEO** [MLWave/kepler-mapper#50]
It would be really helpful for interactive analysis if the tooltip labels could be strings instead of just integer labels. Otherwise one has to reference their original label mapping to understand which integers correspond to what labels/categories. If this functionality is already implemented please demonstrate. Thank you.

**Comment by MLWave** [comment]
You can pass it a numpy array of strings.

```Python
import numpy as np
import kmapper as km
from sklearn import datasets

X, y = datasets.make_circles(5000)

y = np.where(y > 0.5, 'One', 'Zero')

mapper = km.KeplerMapper()

projected_X = mapper.fit_transform(X, projection="dist_mean")
G = mapper.map(projected_X, inverse_X=X)

mapper.visualize(G, path_html="with_string_tooltips.html", custom_tooltips=y)
```

![text labels](https://i.imgur.com/gVKOhqU.png)
